src/data/faq.json

```diff
@@ -1582,10 +1582,11 @@
                                 "active": false,
                                 "textblock": [
-                                    "A backup of the old random IDs is unfortunately not possible, because the Exposure Notification Framework of Apple/Google does not allow the export of this data. To still be informed about possible risk encounters, we recommend that you keep a close view of the risk status on the old device for at least 14 days after you install the Corona-Warn-App on the new device. Please note the following instructions:",
+                                    "A backup of the old random IDs is unfortunately not possible, because the Exposure Notification Framework of Apple/Google does not allow the export of this data. To still be informed about possible risk encounters, we recommend that you keep a close view of the risk status on the old device for at least 14 days (10 days since Version 2.20) after you install the Corona-Warn-App on the new device. Please note the following instructions:",
                                     "<ul><li>Your old device must be able to perform regular checks for possible encounters. Therefore, leave the device permanently turned on and permanently connected to the Internet.</li><li>Keep the old device in a stationary location, e.g., in a specific place in your apartment. If possible, place the device where you can hear or see it so that you are alerted early by a notification of the app.</li><li>Actively access the Corona-Warn-App on the old device at least once a day and check your risk status there.</li></ul>",
-                                    "If you are tested for COVID-19 during these 14 days, enter the PCR-Test QR code on your new device. If the test result is positive, you should share the test result on both devices. For sharing a test result on multiple devices, please note the FAQ entry <a href='#test_multiple_devices' target='_blank' >How do I share a positive test result on multiple devices?</a>.",
-                                    "After 14 days, your old device will no longer contain any relevant entries for risk assessment. You don't need to pay any further attention to the risk status on this device and can remove the Corona-Warn-App there."
+                                    "If you are tested for COVID-19 during these 14 days (10 days since Version 2.20), enter the PCR-Test QR code on your new device. If the test result is positive, you should share the test result on both devices. For sharing a test result on multiple devices, please note the FAQ entry <a href='#test_multiple_devices' target='_blank' >How do I share a positive test result on multiple devices?</a>.",
+                                    "If you are using another compatible COVID-19 app which holds more than 10 days on your old device, you should keep the old device for 14 days regardless of your version of the Corona-Warn-App on your new device.",

```

Comment:
I did not yet translate this into German, as I'm unsure if this is necessary.

Feedback from @MikeMcC399 and the Open-Source-Team is welcome!

Reply:
@Ein-Tim 

I think this should be left out. I will post my reasons in the issue. Thanks for questioning this!

Reply:
I can handle this deletion if there aren't any objections.